content/farci-tomatoes.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Farcies tomatoes
+tags: ['french', 'tomatoes', 'oven']
+date: 2022-06-22
```

Comment:
and here, you should put 2022-07-08 otherwise it will not show at the top of the page

Reply:
Ah thanks! I wrote it sometime ago and forgot to update the date. Sorry!